doc/source/api/general/fielddata.rst

```diff
@@ -18,7 +18,7 @@ by reading case and data files).
   >>> import ansys.fluent.core as pyfluent
   >>> from ansys.fluent.core import examples
   >>> import_file_name = examples.download_file("mixing_elbow.msh.h5", "pyfluent/mixing_elbow")
-  >>> solver = pyfluent.launch_fluent(mode="solver")
+  >>> solver = pyfluent.launch_fluent(mode=pyfluent.FluentMode.SOLVER)
```

Comment:
```suggestion
  >>> solver = pyfluent.launch_fluent()
```

Reply:
We will reassess all these separately - I have added it as an item in the epic #2412